polars/tutorial_dataframe_transformer.py

```diff
@@ -0,0 +1,217 @@
+def _(mo):
+    mo.md(
+        r"""
+        ## Reference a pl.DataFrame
+        Let's reference the same frame as before, but this time as a `pl.DataFrame` by calling `.collect()` on it.
+        """
+    )
+    return
+
+
+@app.cell
+def _(demand):
+    demand.collect()
+    return
+
+
+@app.cell
+def _(mo):
+    mo.md(
+        r"""
+        Note how much functionality we have right out-of-the-box. Click on column names to see rich features like sorting, freezing, filtering, searching, and more!
+
+        Notice how `order_quantity` has a green bar chart under it indicating the ditribution of values for the field!
+
+        Don't miss the `Download` feature as well which supports downloading in CSV, json, or parquet format!
+        """
+    )
+    return
+
+
+@app.cell
+def _(mo):
+    mo.md(
+        r"""
+        ## Use `mo.ui.table`
+        The `mo.ui.table` allows you to select rows for use downstream. You can select the rows you want, and then use these as filtered rows downstream.
+        """
+    )
+    return
+
+
+@app.cell
+def _(demand, mo):
+    demand_table = mo.ui.table(demand, label="Demand Table")
+    return (demand_table,)
+
+
+@app.cell
+def _(demand_table):
+    demand_table
+    return
+
+
+@app.cell
+def _(mo):
+    mo.md(r"""I like to use this feature to select groupings based on summary statistics so I can quickly explore subsets of categories. Let me show you what I mean.""")
+    return
+
+
+@app.cell
+def _(demand, pl):
+    summary: pl.LazyFrame = demand.group_by("product_family").agg(
+        pl.mean("order_quantity").alias("mean"),
+        pl.sum("order_quantity").alias("sum"),
+        pl.std("order_quantity").alias("std"),
+        pl.min("order_quantity").alias("min"),
+        pl.max("order_quantity").alias("max"),
+        pl.col("order_quantity").null_count().alias("null_count"),
+    )
+    return (summary,)
+
+
+@app.cell
+def _(mo, summary):
+    summary_table = mo.ui.table(summary)
+    return (summary_table,)
+
+
+@app.cell
+def _(summary_table):
+    summary_table
+    return
+
+
+@app.cell
+def _(mo):
+    mo.md(
+        r"""
+        Now, instead of manually creatinga filter for what I want to take a closer look at, I simply select from the ui and do a simple join to get that aggregated level with more detail.
+
+        The following cell uses the output of the `mo.ui.table` selection, selects its unique keys, and uses that to join for the selected subset of the original table.
+        """
+    )
+    return
+
+
+@app.cell
+def _(demand, pl, summary_table):
+    selection_keys: pl.LazyFrame = (
+        summary_table.value.lazy().select("product_family").unique()
+    )
+    selection: pl.lazyframe = selection_keys.join(
+        demand, on="product_family", how="left"
+    )
+    selection.collect()
+    return
+
+
+@app.cell
+def _(mo):
+    mo.md(r"""## Use `mo.ui.dataframe`""")
+    return
+
+
+@app.cell
+def _(demand, mo):
+    mo_dateframe = mo.ui.dataframe(demand.collect())
```

Comment:
Should this be mo_dataframe? Wondering if the naming is intentional.
```suggestion
    mo_dataframe = mo.ui.dataframe(demand.collect())
```